app/scripts/metamask-controller.js

```diff
@@ -6009,21 +5928,22 @@ export default class MetamaskController extends EventEmitter {
           this.networkController.upsertNetworkConfiguration.bind(
             this.networkController,
           ),
-        setActiveNetwork: async (networkClientId) => {
-          await this.networkController.setActiveNetwork(networkClientId);
+        // seems like we should make this a noop in the multichain flow
+        setActiveNetwork: async () => {
+          // await this.networkController.setActiveNetwork(networkClientId);
           // if the origin has the eth_accounts permission
           // we set per dapp network selection state
-          if (
-            this.permissionController.hasPermission(
-              origin,
-              PermissionNames.eth_accounts,
-            )
-          ) {
-            this.selectedNetworkController.setNetworkClientIdForDomain(
-              origin,
-              networkClientId,
-            );
-          }
+          // if (
+          //   this.permissionController.hasPermission(
+          //     origin,
+          //     PermissionNames.eth_accounts,
+          //   )
+          // ) {
+          //   this.selectedNetworkController.setNetworkClientIdForDomain(
+          //     origin,
+          //     networkClientId,
+          //   );
+          // }
```

Comment:
unsure... I suppose the active network will still be a thing for a bit though, so maybe we just leave this as is. Need to verify impact again